### DISCUSSION TITLE
What input shaping settings and speed/accel has everyone settled on?

### DISCUSSION THREAD

**Discussion by confuzled-dev** [dz0ny/klipper-prusa-mk3s#34] [General]
Curious to see some input-shaping graphs for these Prusa printers. I am running 3100 accel, 200 mm/s infill, 90 mm/s ext perimeters, and 50 small perimeters. Curious how everyone else is getting along.

**Comment by confuzled-dev** [comment]
Hoping to bump this. We all have the same motors, same frame, and relatively same beds. 

Has anyone done any tmc stepper tuning? What about VFAs, any noticeable increase/decrease relative to stock?

**Comment by confuzled-dev** [comment]
I am getting pretty pronounced VFA and am reverting back to prior to the linear correction change to see if this caused an issue. Has anyone else seen an increased appearance of VFA with the new linear correction feature (set to 0 as default)

**Reply by agravelot** [reply]
Too much vfa mostly means too much tension on your belts

**Reply by confuzled-dev** [reply]
I notice that I have specific range that has bad VFA. If  I avoid those speeds, it is much better

**Comment by confuzled-dev** [comment]
![image](https://github.com/dz0ny/klipper-prusa-mk3s/assets/4993555/a53af4c2-8a71-4fa5-8fd8-81edf526371c)
Here is what I am experiencing

**Reply by b-fitzpatrick** [reply]
@dougshell About a 2mm pitch? I have a similar issue at speeds that correspond with the belt pitch of 2mm and resonant frequency of the Y-axis. It resonated at around 37 Hz (when in the middle of y-travel), so the issue was worst at ~74 mm/s. A toothed idler was making it worse, especially in one direction. It improved with the stock, smooth idler, but is still there. I designed some stiffeners for the front and rear plates that moved the resonance frequency up to 40 Hz, which allowed me to run a bit faster before running into this. I have some 10mm-wide belt and pulleys that I'm going to try to raise the resonance frequency higher.

**Reply by b-fitzpatrick** [reply]
In case anyone comes across this, a 10mm-wide belt did absolutely nothing to the Y-axis resonant frequency. I knew the belts were stiff, but I figured it would go up a bit. Something else in the system is still much, much softer. I should have done some more research before going down that path.

Before (6mm belt):
![image](https://github.com/dz0ny/klipper-prusa-mk3s/assets/9221473/650c5db8-a0a7-4cf1-af6a-334bd551f31a)

After (10mm belt):
![image](https://github.com/dz0ny/klipper-prusa-mk3s/assets/9221473/84d93c7d-565a-49fb-96a6-d56eab4200ee)

The accelerometer is mounted at the middle of the right side of the bed for both tests (25-75 Hz sweeps). The printer is on stock rubber feet pushed to the extreme front/rear and on a fairly heavy bench. I don't run it that way, but that's how I tested it last.

**Reply by b-fitzpatrick** [reply]
To check the "stiffness" of the motor, I increased the Y-axis motor current from 0.305 to 0.505A. It raised the resonant frequency about 4 Hz (and caused a bit more vibration at higher frequencies). I'm not planning to leave it there.
![image](https://github.com/dz0ny/klipper-prusa-mk3s/assets/9221473/38c3780c-f337-4af6-8ad4-4a01f36e8234)

**Comment by dz0ny** [comment]
Hey @dougshell the way discussions work on GitHub mostly only those who contributed to the repo will see the messages. I suggest you join the Klipper community on Discord as more users should be available.

**Reply by confuzled-dev** [reply]
Oh, I thought everyone who had previously discussed would see it. Ok. Sorry for all the chatter. Just trying to get things working right. I am talking there now

**Comment by confuzled-dev** [comment]
I have improved the issue by increasing internal perimeter and lowering
external.

I'm in a much better place quality wise

**Reply by TheLion** [reply]
What are the speeds/acceleration you settled on?

**Comment by confuzled-dev** [comment]
I think it is better to print the bed mount with a large brim and resizes
holes and mount the adxl straight to the bed in the center.

**Reply by b-fitzpatrick** [reply]
Yes, I've printed a mount in the center of the bed, but I found mounting it on the side works pretty well. It's certainly good enough for comparison testing.
